### PR TITLE
Loginform

### PR DESCRIPTION
**Pull request by Liliya-V**

### CODE REVIEW

File: adminPanel/login.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+define ('USERNAME', 'Liliya_portfolio');
+define ('Password', '$2y$10$CudgPciI3HI7muiuYmX9c.pxj9e7dSiij1LX3T8CVyA4mlC0KOsei');
+if (!empty($_POST['username']) &&
+    !empty($_POST['password'])) {
+    $username = $_POST['username'];
+    $password = $_POST['password'];
+    SESSION_START();
+    if ($username == USERNAME &&
```
Comment (alecorra) [inline adminPanel/login.php]: == is ok but it's better if they are of the same type (===)

**Review by alecorra — COMMENTED** [review]
very good, just only small suggestion.

File: adminPanel/adminPanel.php [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <?php
+session_start();
+if  (empty($_SESSION['loggedIn']) ||
```
Comment (mporam) [inline adminPanel/adminPanel.php]: not PSR2. please put the if on one line:
```
if (condition || condition) {
```

File: adminPanel/login.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+define ('USERNAME', 'Liliya');
+define ('PASSWORD', '$2y$10$nndShKCDK2/qokmFytHNsuajSG5PxtY7gICsr6V6IDY92dFxHWIqG');
+if (!empty($_POST['username']) &&
```
Comment (mporam) [inline adminPanel/login.php]: put all conditions on one line please

File: adminPanel/loginform.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php
+if (!empty($_GET['login'])) {
+    echo 'Username or password is incorrect';
+}
+?>
+
+<form action="login.php" method="POST">
```
Comment (mporam) [inline adminPanel/loginform.php:16]: no doctype, html tag or body tag?

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**